tests/checkbox/checkbox-provider-tdx/units/basic/jobs.pxu

```diff
@@ -83,6 +83,18 @@ command:
   export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/:$LD_LIBRARY_PATH
   setup-env-and-run test_boot_multiple_vms.py
 
+id: tdx-basic/td-stress-boot
+category_id: tdx-basic
+flags: simple
+_summary: Test boot stress
+depends:
+after:
+requires:
+  executable.name == 'qemu-system-x86_64'
+command:
+  export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/:$LD_LIBRARY_PATH
```

Comment:
Is there any way in checkbox to define this command at the unit scope (or wherever) so you don't need to run it for each test?

Reply:
This is something that already exists before this PR, we can discuss about this for sure but not in the scope of this MR, but however, here is my comment: for this, AFAIK, i think there is only 2 scopes : global and test. No possibility to have a globe that covers category or family/set of tests.